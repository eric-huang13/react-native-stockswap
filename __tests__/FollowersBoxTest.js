import React from 'react';
import FollowersBox from '../src/components/HomeTabComponents/FollowersBox';
import ShallowRenderer from 'react-test-renderer/shallow'; // ES6

jest.mock('@react-native-community/google-signin', () => {});

jest.mock("@react-native-async-storage/async-storage", () =>
  require("@react-native-async-storage/async-storage/jest/async-storage-mock"),
);
jest.mock('react-redux', () => ({
    connect: () => {
      return (component) => {
        return component
      };
    }
  }));

  test('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FollowersBox item={{img:'test' }}/>);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });