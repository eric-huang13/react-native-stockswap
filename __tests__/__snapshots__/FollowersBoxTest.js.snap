// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ForwardRef(SafeAreaView)
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2a334a",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingHorizontal": 14.124087591240876,
      "paddingVertical": 19.77372262773723,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "test",
        }
      }
      style={
        Object {
          "borderRadius": 70.62043795620437,
          "height": 88.98175182481751,
          "width": 88.98175182481751,
        }
      }
    />
    <View
      style={
        Object {
          "marginLeft": 8.474452554744525,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontFamily": "Montserrat-Bold",
            "fontSize": 22.598540145985403,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "lightgrey",
            "fontFamily": "Montserrat-Regular",
            "fontSize": 19.77372262773723,
          }
        }
      />
    </View>
  </View>
</ForwardRef(SafeAreaView)>
`;
